service_management/azure/test/integration/service_bus/informative_errors_test.rb

```diff
@@ -13,18 +13,23 @@
 # limitations under the License.
 #--------------------------------------------------------------------------
 require "integration/test_helper"
-require "azure/blob/blob_service"
 require "azure/core/http/http_error"
 
-describe "ServiceBus errors" do
+describe Azure::ServiceBus::ServiceBusService do
```

Comment:
I saw that all our other ASM integration test files have the pattern of having first describe name as the module/service name for which the test is being written, so that i updated it to be consistent.